### PR TITLE
replace no-remove by sample-volume-plugin in docs

### PR DESCRIPTION
**Pull request by vieux**
ping @anusha-ragunathan @tiborvass

### CODE REVIEW

**Comment by tiborvass** [comment]
LGTM

File: docs/reference/commandline/plugin_install.md [hunk 1]
```diff
@@ -32,20 +32,20 @@ the registry. Note that the minimum required registry version to distribute
...
 the user to accept the list of privileges that the plugin needs, settings parameters
  and enabling the plugin.
 
 ```bash
-$ docker plugin install tiborvass/no-remove DEBUG=1
+$ docker plugin install tiborvass/sample-volume-plugin DEBUG=1
 
-Plugin "tiborvass/no-remove" is requesting the following privileges:
+Plugin "tiborvass/sample-volume-plugin" is requesting the following privileges:
  - network: [host]
```
Comment (anusha-ragunathan) [inline docs/reference/commandline/plugin_install.md]: This privilege set is not what `sample-volume-plugin` requests. Please update.

Comment (vieux) [inline docs/reference/commandline/plugin_install.md]: @there is not privileges in `tiborvass/sample-volume-plugin` it doesn't even ask the question.

Should I use a fake plugin, that doesn't exists ? or mine ?

File: docs/extend/config.md [hunk 2]
```diff
@@ -171,52 +171,49 @@ Config provides the base accessible fields for working with V0 plugin format
...
-  "linux": {
-    "devices": [
-      {
-        "name": "device",
-        "description": "a host device to mount",
-        "path": "/dev/cpu_dma_latency"
-      }
-    ]
-  }
+            "Args": {
```
Comment (thaJeztah) [inline docs/extend/config.md]: Did this all change from lowercase to uppercase? Is that intentional?

File: api/swagger.yaml [hunk 3]
```diff
@@ -6361,74 +6326,39 @@ paths:
...
                   Workdir: ""
                   User: {}
                   Network:
-                    Type: "host"
-                  Capabilities: null
-                  Mounts:
-                    - Name: ""
-                      Description: ""
-                      Settable: null
-                      Source: "/data"
```
Comment (thaJeztah) [inline api/swagger.yaml]: Wondering if we can have output that has more options set, to show what users can expect as options

File: docs/reference/commandline/plugin_install.md (hunk 1 above)
Comment (anusha-ragunathan) [inline docs/reference/commandline/plugin_install.md]: Use yours.

Comment (anusha-ragunathan) [inline docs/reference/commandline/plugin_install.md]: Actually, `sample-volume-plugin` has a `PropagatedMount` which should prompt privileges. There's a bug somewhere.

File: docs/extend/config.md (hunk 2 above)
Comment (vieux) [inline docs/extend/config.md]: it's case incensitive I used what's returned by `docker plugin inspect`

File: docs/reference/commandline/plugin_install.md (hunk 1 above)
Comment (tiborvass) [inline docs/reference/commandline/plugin_install.md]: Let's use `vieux/ssh` instead.

**Review by thaJeztah — APPROVED** [review]
LGTM (for now :smile:)

**Pull request merged**